### PR TITLE
[FLINK-18282][docs-zh] retranslate the documentation home page

### PR DESCRIPTION
**Pull request by sdlcwangsong**
## What is the purpose of the change

FLINK-17981 was a complete rewrite of the documentation home page. The chinese translation should be updated along the same lines.

## Brief change log

- retranslate `flink/docs/index.zh.md`

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: no
  - The serializers:no
  - The runtime per-record code paths (performance sensitive): no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn/Mesos, ZooKeeper: no
  - The S3 file system connector: no

## Documentation

  - Does this pull request introduce a new feature? no
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

File: docs/index.zh.md [hunk 1]
```diff
@@ -23,53 +23,71 @@ specific language governing permissions and limitations
...
-## API 参考
+* [概念透析]({% link concepts/index.md %}) 介绍了在浏览参考文档之前你需要了解的 Flink 知识。
 
-API 参考列举并解释了 Flink API 的所有功能。
+### 获取 Flink 帮助
 
-* [DataStream API](dev/datastream_api.html)
-* [DataSet API](dev/batch/index.html)
-* [Table API &amp; SQL](dev/table/index.html)
+如果你被困住了, 可以在 [社区](https://flink.apache.org/community.html)寻求帮助。 值得一提的是，Apache Flink 的用户邮件列表一直是最活跃的 Apache 项目之一，也是一个快速获得帮助的好途径。
```
Comment (libenchao) [inline docs/index.zh.md:50]:
```suggestion
如果你遇到困难了, 可以在 [社区](https://flink.apache.org/community.html)寻求帮助。值得一提的是，Apache Flink 的用户邮件列表一直 Apache 项目是最活跃的之一，也是一个快速获得帮助的好途径。
```

File: docs/index.zh.md [hunk 2]
```diff
@@ -23,53 +23,71 @@ specific language governing permissions and limitations
...
-Apache Flink 是一个分布式流批一体化的开源平台。Flink 的核心是一个提供数据分发、通信以及自动容错的流计算引擎。Flink 在流计算之上构建批处理，并且原生的支持迭代计算，内存管理以及程序优化。
+### 试用 Flink
 
-## 初步印象
+如果您有兴趣使用 Flink, 可以试试我们的教程:
 
-* **代码练习**: 跟随分步指南通过 Flink API 实现简单应用或查询。
-  * [实现 DataStream 应用]({% link try-flink/datastream_api.zh.md %})
-  * [书写 Table API 查询]({% link try-flink/table_api.zh.md %})
+* [DataStream API 进行欺诈检测]({% link try-flink/datastream_api.md %})
```
[the remark this answers is not in the document]
Comment (libenchao) [reply docs/index.zh.md:37]: same for all the links in this doc.

File: docs/index.zh.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Apache Flink 文档"
+title: "Apache Flink Documentation"
```
[the remark this answers is not in the document]
Comment (sdlcwangsong) [reply docs/index.zh.md:2]: thanks, I will keep the original translation here

File: docs/index.zh.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (sdlcwangsong) [reply docs/index.zh.md:37]: I think this is a relative path, so no modification

File: docs/index.zh.md [hunk 4]
```diff
@@ -23,53 +23,71 @@ specific language governing permissions and limitations
...
-* **Docker 游乐场**: 你只需花几分钟搭建 Flink 沙盒环境，就可以探索和使用 Flink 了。
-  * [运行与管理 Flink 流处理应用]({% link try-flink/flink-operations-playground.zh.md %})
+### 学习 Flink
 
-* **概念**: 学习 Flink 的基本概念能更好地理解文档。
-  * [有状态流处理](concepts/stateful-stream-processing.html)
-  * [实时流处理](concepts/timely-stream-processing.html)
-  * [Flink 架构](concepts/flink-architecture.html)
-  * [术语表](concepts/glossary.html)
+* [操作培训]({% link learn-flink/index.md %}) 包含了一系列的课程和练习，逐步介绍了，帮助你深入学习 Flink。
```
[the remark this answers is not in the document]
Comment (sdlcwangsong) [reply docs/index.zh.md:44]: I will complete this

File: docs/index.zh.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (libenchao) [reply docs/index.zh.md:37]: You can see the discussions here: http://apache-flink-mailing-list-archive.1008284.n3.nabble.com/Reminder-Prefer-link-tag-in-documentation-tt42362.html#none

**Comment by sdlcwangsong** [comment]
hi, @libenchao, I have finish it

File: docs/index.zh.md (hunk 1 above)
Comment (libenchao) [reply docs/index.zh.md]: I think you maybe misunderstood my comment here. others LGTM

File: docs/index.zh.md [hunk 5]
```diff
@@ -23,53 +23,71 @@ specific language governing permissions and limitations
...
-* **Docker 游乐场**: 你只需花几分钟搭建 Flink 沙盒环境，就可以探索和使用 Flink 了。
-  * [运行与管理 Flink 流处理应用]({% link try-flink/flink-operations-playground.zh.md %})
+### 学习 Flink
 
-* **概念**: 学习 Flink 的基本概念能更好地理解文档。
-  * [有状态流处理](concepts/stateful-stream-processing.html)
-  * [实时流处理](concepts/timely-stream-processing.html)
-  * [Flink 架构](concepts/flink-architecture.html)
-  * [术语表](concepts/glossary.html)
+* [操作培训]({% link learn-flink/index.zh.md %}) 包含了一系列的课程和练习，提供了对Flink的逐一介绍。
```
Comment (libenchao) [inline docs/index.zh.md:44]:
```suggestion
* [操作培训]({% link learn-flink/index.zh.md %}) 包含了一系列的课程和练习，提供了对 Flink 的逐一介绍。
```

Comment (sdlcwangsong) [reply docs/index.zh.md]: @libenchao， thanks for you review

File: docs/index.zh.md (hunk 1 above)
Comment (sdlcwangsong) [reply docs/index.zh.md]: yes, I made some modify. @libenchao

**Pull request closed without merging**